src/Features/Core/Portable/CodeFixes/Configuration/ConfigurationUpdater.cs

```diff
@@ -185,9 +185,25 @@ private async Task<Solution> ConfigureAsync()
             // Compute the updated text for analyzer config document.
             var newText = GetNewAnalyzerConfigDocumentText(originalText, editorConfigDocument);
 
-            return newText != null
-                ? solution.WithAnalyzerConfigDocumentText(editorConfigDocument.Id, newText)
-                : solution;
+            if (newText == null)
+            {
+                return solution;
+            }
+
+            // Add the new added analyzer config document as a solution item.
+            if (PathUtilities.IsAbsolute(editorConfigDocument.FilePath) &&
+                !File.Exists(editorConfigDocument.FilePath))
```

Comment:
FYI, ops like File.Exists can throw.  Is this codepath resilient to htat?

Reply:
Nope, I should handle possibly exception here and also in one other place I used it in this PR.